test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+docker compose up --wait localstack
+docker-compose exec -T localstack bash -c '. /scripts/wait/wait-until-s3-ready.sh'
+docker-compose exec -T localstack bash -c 'echo "Test file" | awslocal s3 cp - s3://uploads-bucket/valid.txt'
+docker-compose exec -T localstack bash -c 'echo "X5O!P%@AP[4\PZX54(P^)7CC)7}\$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!\$H+H*" | awslocal s3 cp - s3://uploads-bucket/invalid.txt'
```

Comment:
cool, I was hoping there would be a nice test value

Reply:
I can't wait for Veracode to false-positive us for having a virus in the repo 😅